cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/ClusterBootstrapSettings.scala

```diff
@@ -13,43 +13,33 @@ import com.typesafe.config.Config
 import scala.concurrent.duration.{ FiniteDuration, _ }
 
 final class ClusterBootstrapSettings(config: Config) {
-  private implicit class HasDefined(val config: Config) {
-    def hasDefined(key: String): Boolean =
-      config.hasPath(key) &&
-      config.getString(key).trim.nonEmpty &&
-      config.getString(key) != s"<$key>"
-  }
+  import akka.management.AkkaManagementSettings._
 
   val managementBasePath: Option[String] =
     Option(config.getString("akka.management.http.base-path")).filter(_.trim.nonEmpty)
 
   private val bootConfig = config.getConfig("akka.management.cluster.bootstrap")
 
-  val formNewCluster: Boolean = bootConfig.getBoolean("form-new-cluster")
+  val newClusterEnabled: Boolean = bootConfig.getBoolean("new-cluster-enabled")
```

Comment:
The new property name is much more clear, but I guess this will do the wrong thing on upgrade for anyone who is setting the old property and doesn't notice that it changed. Maybe we could detect the old property name, and if it's present, use that and warn?

Reply:
As a special case of that, the old property name is used in the Lightbend Orchestration tooling: https://github.com/lightbend/reactive-cli/search?q=%22form-new-cluster%22

Reply:
Hrm, I'll wait for @patriknw or @chbatey's input on preference for proceeding on this one. Good point.

Reply:
I agree that we should detect if the old property is set and then fail immediatly at startup, directing users to the new property

```
bootConfig.hasPath("form-new-cluster")
```

Reply:
after some additional clarifications from @TimMoore I understand that we can't fail fast if the old property was set since that would break reactive-cli.

We can change reactive-cli to use the new property but there is anyway a risk that the old reactive-cli version is used with this newer version of bootstrap, so at least for a migration period we need to be backwards compatible.

We need to read both properties. If `form-new-cluster=off` then we should use `off`, otherwise we can use the value of the new property `new-cluster-enabled`. Does that cover it?